src/test/java/io/candydoc/plugin/save_ubiquitous_language/file/AssertFileContent.java

```diff
@@ -1,40 +1,60 @@
 package io.candydoc.plugin.save_ubiquitous_language.file;
 
-import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.jupiter.api.Assertions.fail;
 
 import com.fasterxml.jackson.core.JsonFactory;
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import java.io.IOException;
 import java.nio.file.Path;
-import lombok.AccessLevel;
-import lombok.RequiredArgsConstructor;
+import org.assertj.core.api.AbstractAssert;
 
-@RequiredArgsConstructor(access = AccessLevel.PRIVATE)
-public class AssertFileContent {
+public class AssertFileContent extends AbstractAssert<AssertFileContent, Path> {
 
-  private final Path actualPath;
-  private final ObjectMapper trustedMapper;
+  private ObjectMapper trustedMapper;
 
-  public AssertFileContent isEqualTo(Path expectedPath) {
-    try {
-      JsonNode actualJson = trustedMapper.readTree(actualPath.toFile());
-      JsonNode expectedJson = trustedMapper.readTree(expectedPath.toFile());
+  protected AssertFileContent(Path actual) {
+    super(actual, AssertFileContent.class);
+  }
 
-      assertThat(actualJson).isEqualTo(expectedJson);
-    } catch (Exception e) {
-      fail(e);
+  public static AssertFileContent assertThat(Path actual) {
+    return new AssertFileContent(actual);
+  }
+
+  public AssertFileContent exists() {
+    isNotNull();
+    if (!actual.toFile().exists()) {
+      failWithMessage("File %s doesn't exists", actual);
     }
```

Comment:
what about using the `assertThat(path).exists()` already existing from AssertJ instead of this condition ?